File: app/src/main/java/se/appshack/android/refactoring/main/data/model/GenusResponseModel.java
package se.appshack.android.refactoring.main.data.model;

import com.google.gson.annotations.SerializedName;

public class GenusResponseModel {

    @SerializedName("genus")
    public String genus;

    @SerializedName("language")
    public NamedResponseModel language;
}

File: app/src/main/java/se/appshack/android/refactoring/main/data/model/GenusResponseModel.kt
package se.appshack.android.refactoring.main.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class GenusResponseModel (

    @SerializedName("genus")
    var genus: String? = null,

    @SerializedName("language")
    var language: NamedResponseModel? = null

): Parcelable
